src/Converters/Converters.Tests/Mocks/MockConverter.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace Converters.Tests.Mocks
+{
+    public class MockConverter : IValueConverter
```

Comment:
Please move the converter tests to the sample application, instead inline in the general library ( to keep the actual nuget result small )

Reply:
Ok, but why? tests are not contained in the resulting Nuget package:
https://github.com/FormsCommunityToolkit/FormsCommunityToolkit/blob/dev/nuget/Converters.nuspec#L27
only general assembly of converters

Reply:
Are they stripped? Ok never mind then, did not double check the nuget, was something James has set up.